airflow/serialization/serialized_objects.py

```diff
@@ -432,6 +437,9 @@ def deserialize_operator(cls, encoded_op: Dict[str, Any]) -> BaseOperator:
             if not hasattr(op, field):
                 setattr(op, field, None)
 
+        # Used to determine if an Operator is inherited from DummyOperator
+        setattr(op, "_is_dummy", bool(encoded_op.get("_is_dummy")))
```

Comment:
```suggestion
        setattr(op, "_is_dummy", bool(encoded_op.get("_is_dummy", False)))
```

To load existing s10n blobs.

Reply:
fixed in https://github.com/apache/airflow/pull/12745/commits/67b5daf93d7a791670966cea806429155a303041